### PR TITLE
PS-290: Added testcase (5.5)

### PR DESCRIPTION
**Pull request by dutow**

### CODE REVIEW

**Comment by dutow** [comment]
https://jenkins.percona.com/view/PS%205.5/job/percona-server-5.5-param/1633/

File: mysql-test/suite/binlog/t/percona_bug_ps290-master.opt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+--log-bin
```
Comment (laurynas-biveinis) [inline mysql-test/suite/binlog/t/percona_bug_ps290-master.opt]: Is the option file required?

**Review by laurynas-biveinis — COMMENTED** [review]
Approved if the option file is indeed required, otherwise approved with it removed

File: mysql-test/suite/binlog/t/percona_bug_ps290-master.opt (hunk 1 above)
Comment (dutow) [reply mysql-test/suite/binlog/t/percona_bug_ps290-master.opt]: No, looks like it's not needed, I removed the file.

**Review by laurynas-biveinis — APPROVED** [review]

**Pull request merged**